cli/dcoscli/package/main.py

```diff
@@ -58,7 +58,9 @@ def _cmds():
 
         cmds.Command(
             hierarchy=['package', 'describe'],
-            arg_keys=['<package_name>', '--cli', '--app', '--options'],
+            arg_keys=['<package_name>', '--cli', '--app', '--options',
```

Comment:
A slight concern - for consistency, it would be good to have --app, then --cli (to match the docopt package.txt above).